Makefile

```diff
@@ -33,6 +33,7 @@ __package-sbt:
 		coverageReport \
 		coverallsMaybe \
 		coverageOff \
+		"set coverageEnabled := false" \
```

Comment:
note; the last container published failed to start because coverage was not really turned off which published a binary wish scoverage invoker instrumentation present